astropy/convolution/tests/test_convolve_models.py

```diff
@@ -6,9 +6,9 @@
 import pytest
 from numpy.testing import assert_allclose, assert_almost_equal
 
-from astropy.convolution.convolve import convolve, convolve_fft, convolve_models
+from astropy.convolution.convolve import convolve, convolve_fft, convolve_models  # noqa: F401
```

Comment:
Just fixed a broken test by adding `convolve` and `convolve_fft` back in to the file with the `noqa`.

This is because they are used when evaluating the string in this part of the tests: https://github.com/astropy/astropy/blob/f9f71e3e170290c3e3beb4dd2385dde8ac1733d4/astropy/convolution/tests/test_convolve_models.py#L15-L23

Should the `eval` be converted directly into explicit python code?

Reply:
YES! 😭